en/tutorials/index.md

```diff
@@ -11,7 +11,7 @@ If you experience any problems at any step of this doc, please check our <a href
 
 ### Prerequisites
 
-You should have [installed Hoodie and its prerequisites](/en/start/) and read the [first part of this guide](/en/start/getting-started/getting-started-1.html). We'll be looking at the Hoodie app you created in that.
+You should have [installed Hoodie and its prerequisites](/en/start/) and read the [first part of this guide](/en/start/getting-started/getting-started-1.md). We'll be looking at the Hoodie app you created in that.
```

Comment:
> `[first part of this guide](/en/start/getting-started/getting-started-1.md)`

This link was pointing to an `.html` file before because these Markdown files get converted to HTML for the Docs website. It will need to be reverted back to `.html` so the link still works.